data-reporting-service/src/main/java/gov/cdc/etldatapipeline/changedata/model/dto/persondetail/AddAuthUser.java

```diff
@@ -0,0 +1,37 @@
+package gov.cdc.etldatapipeline.changedata.model.dto.persondetail;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import gov.cdc.etldatapipeline.changedata.model.dto.PersonExtendedProps;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AddAuthUser {
+    @JsonProperty("ADD_USER_ID")
+    private Long addUserId;
+    @JsonProperty("add_time")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
```

Comment:
@rseshadri99 @upasanapattnaik-eq Deserializing the date time in the local timezone as discussed in slack.